paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -76,6 +76,7 @@ public class PaparazziPlugin : Plugin<Project> {
   private fun <T> setupPaparazzi(project: Project, variants: DomainObjectSet<T>) where T : BaseVariant, T : TestedVariant {
     project.addTestDependency()
     val nativePlatformFileCollection = project.setupNativePlatformDependency()
+    val layoutlibResourcesFileCollection = project.setupLayoutlibResourcesDependency()
```

Comment:
Everything except for the native lib itself moved to a new `layoutlib-resources` artifact in 14.0.3, so we need to unzip that one as well and have 2 system properties for passing the directories to the test